README.md

```diff
@@ -1,2 +1,27 @@
-# rdk-hpk-documentation
-This repo holds public documentation to support the HPK (Hardware Porting Kit)
+# HPK Public Documentation
+
+## Table of Contents
+
+- [Acronyms, Terms and Abbreviations](#acronyms-terms-and-abbreviations)
+- [Description](#description)
+- [Hardware Porting Kit](#hardware-porting-kit)
+- [Release History](#release-history)
+
+## Acronyms, Terms and Abbreviations
+
+- `HPK` - Hardware Porting Kit
+- `HAL` - Hardware Abstraction Layer
+- `SOC` - System on a Chip
+- `OEM` - Original Equipment Manufacturer
+
+## Description
+
+This is the public documentation to support the Hardware Porting Kit (`HPK`), for any vendor / third party to read. This document will be the consolidating jump point for all documentation with links to other open documentations.
+
+## Hardware Porting Kit
+
+The Hardware Porting Kit (`HPK`) represents an advanced stage in the progression of the extensively delineated Hardware Abstraction Layer (`HAL`). It is enhanced with extensive documentation and thorough test suites, empowering `OEM` or `SOC` vendors to independently validate their ports prior to submission for final integration and deployment with an operator. This kit serves as a comprehensive resource, enabling `OEMs` and/or `SOC` vendors to autonomously undergo certification for their devices.
```

Comment:
The hardware porting kit (HPK) is a collection resources that simplify the process of adapting existing software to run on a new hardware platform. It provides a framework for making the necessary changes to the software to account for the differences between the hardware vendors.

It will provide both Hardware Abstraction Layer (API) Header files, and software tests to ensure compatibility with the RDK Middleware Stack.

Reply:
Changed